src/studio/src/designer/DataModeling/Converter/Json/Strategy/GeneralJsonSchemaAnalyzer.cs

```diff
@@ -16,13 +17,13 @@ public GeneralJsonSchemaAnalyzer() : base()
         }
 
         /// <inheritdoc/>
-        public override JsonSchemaXsdMetadata AnalyzeSchema(JsonSchema schema, Uri uri)
+        public override JsonSchemaXsdMetadata AnalyzeSchema(JsonSchema schema)
         {
             JsonSchema = schema;
             Metadata = new JsonSchemaXsdMetadata
             {
                 SchemaOrigin = "Standard",
-                MessageName = UrlHelper.GetName(uri.ToString()),
+                MessageName = JsonSchema.TryGetKeyword(out XsdRootElementKeyword rootElementKeyword) ? rootElementKeyword.Value : "root",
```

Comment:
I assume "root" shouldn't happen and that this is just a "last fallback"?

Reply:
Correct. If we build the json schema doesn't have that keyword defined we'll produce following (default) xml :
```xml
<root>
  ...
</root>
```
since json doesn't have equivalent of the xml root node